wrappers/darwin.nix

```diff
@@ -18,11 +18,11 @@ let
   cfg = config.programs.nixvim;
   nixvimConfig = config.lib.nixvim.modules.evalNixvim {
     extraSpecialArgs = {
-      defaultPkgs = pkgs;
       darwinConfig = config;
     };
     modules = [
       ./modules/darwin.nix
+      { nixpkgs.pkgs = lib.mkDefault pkgs; }
```

Comment:
We could instead do this in a common `wrappers/modules/*.nix` module:

```nix
{ pkgs, lib, ... }:
{
  programs.nixvim.nixpkgs.pkgs = lib.mkDefault pkgs;
}
```

If imported with a relative path, then the module would benefit from automatic `_file` metadata.

Such a module may also be a starting point for a future hm-stlye "useGlobalPackages" option?

Reply:
Moved to `_shared.nix` for now.